src/encoded/tests/testing_views.py

```diff
@@ -9,6 +9,7 @@
 )
 from ..types.base import paths_filtered_by_status
 from snovault.attachment import ItemWithAttachment
+from encoded.types.base import Item as EncodedItem
```

Comment:
Why did you rename this Class EncodedItem?  Do you also load Snovault.Item?

Reply:
Yeah, this is already loaded above for the other fixtures:
```
from snovault import (
    Item,
    calculated_property,
    collection,
)
```

Also we need to specifically use the encoded Item here to get it to work.